src/test/java/be/syntra/devshop/DevshopBack/testutilities/UserUtils.java

```diff
@@ -2,20 +2,26 @@
 
 import be.syntra.devshop.DevshopBack.entities.User;
 import be.syntra.devshop.DevshopBack.models.UserDto;
+import be.syntra.devshop.DevshopBack.security.entities.UserRole;
 
 import java.util.LinkedList;
 import java.util.List;
 
-public class UserUtils {
-
+import static be.syntra.devshop.DevshopBack.security.entities.UserRoles.ROLE_ADMIN;
+import static be.syntra.devshop.DevshopBack.security.entities.UserRoles.ROLE_USER;
 
+public class UserUtils {
 
     public static User createUser() {
         return User.builder()
+                .id(1L)
                 .firstName("Someone")
                 .lastName("First")
                 .fullName("Someone First")
                 .password("password")
+                .email("test@email.com")
+                .userRoles(List.of(UserRole.builder().name(ROLE_USER.name()).build(),
+                        UserRole.builder().name(ROLE_ADMIN.name()).build()))
```

Comment:
can this be newlined into something more readable?

Reply:
Fixed